pkg/controller/topology_test.go

```diff
@@ -1046,6 +1047,29 @@ func TestPreferredTopologies(t *testing.T) {
 				},
 			},
 		},
+		"allowedTopologies specified: no CSINode": {
```

Comment:
nit: change the other similar test cases to "no CSINode" too

Reply:
I split it up into multiple cases, once for immediate binding and once for delayed binding.
* missing a Node object
* Node object doesn't have topology labels
* missing CSINode object
* CSINode object doesn't have topology keys